### DISCUSSION TITLE
sway 下fcitx5能正常工作吗？

### DISCUSSION THREAD

**Discussion by guyuming76** [fcitx/fcitx5#667] [Q&A] [answered]
我在foot里发现选词窗口不显示，输入内容会有下划线，按空格后有中文出现，可能是哪里设置错了吗？

![20221125_12h44m36s_grim](https://user-images.githubusercontent.com/12045548/203912218-ffc02416-d728-442d-9874-1b1d685f5fdd.png)

**Comment by wengxt** [comment, marked as answer]
能,用  https://aur.archlinux.org/packages/sway-im

**Reply by wengxt** [reply]
还有一个进展缓慢的 pr 不知道何时才会合并 https://github.com/swaywm/sway/pull/5890
